### PR TITLE
Add Feishu 3.6.3

### PR DESCRIPTION
**Pull request by boltomli**
Feishu (Lark) is a tool to create team collaboration experiences.

After making all changes to the cask:

- [X] `brew cask audit --download {{cask_file}}` is error-free.
- [X] `brew cask style --fix {{cask_file}}` reports no offenses.
- [X] The commit message includes the cask’s name and version.
- [X] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [X] Named the cask according to the [token reference].
- [X] `brew cask install {{cask_file}}` worked successfully.
- [X] `brew cask uninstall {{cask_file}}` worked successfully.
- [X] Checked there are no [open pull requests] for the same cask.
- [X] Checked the cask was not [already refused].
- [X] Checked the cask is submitted to [the correct repo].
- [X] Attempted to find an [appcast].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256
[appcast]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md

### CODE REVIEW

File: Casks/feishu.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'feishu' do
```
Comment (vitorgalvao) [inline Casks/feishu.rb]: Does not follow the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).

File: Casks/feishu.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  version '3.6.3'
+  sha256 '8ea622f24938354ec95d79946b21b7f10c7bd36f7198b1290375631f6eaad537'
+
+  # sf3-ttcdn-tos.pstatp.com was verified as official when first introduced to the cask
+  url "https://sf3-ttcdn-tos.pstatp.com/obj/ee-appcenter/Feishu_#{version}.dmg"
```
Comment (vitorgalvao) [inline Casks/feishu.rb]: What methods have you tried to find the `appcast`?

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/feishu.rb (hunk 2 above)
Comment (boltomli) [reply Casks/feishu.rb]: I browsed the website, which didn't advertise change log explicitly. There might be some URL but I would need to find in the app's Content or monitor requests over network. The result will not be quite stable in future anyway.

File: Casks/feishu.rb (hunk 1 above)
Comment (boltomli) [reply Casks/feishu.rb]: Thanks for the comment. I should have raised the question. The name is displayed (and widely referenced by the developer and users) as 'Feishu' in the dmg and after installation. The folder and executable name is Lark, though.

File: Casks/feishu.rb (hunk 2 above)
Comment (vitorgalvao) [reply Casks/feishu.rb]: We’re going to need some kind of appcast. What’s the download page?

Comment (boltomli) [reply Casks/feishu.rb]: The page is <https://www.feishu.cn/download>. I managed to dump the API URL to get all downloads and updated the cask.

Comment (vitorgalvao) [reply Casks/feishu.rb]: Fantastic. That’s ideal. Thank you.

**Pull request merged**